hr_payroll_analysis/tests/test_payroll_analysis.py

```diff
@@ -0,0 +1,233 @@
+        self.analysis_line_model = self.registry('hr.payslip.analysis.line')
+
+        self.context = self.user_model.context_get(self.cr, self.uid)
+
+        cr, uid, context = self.cr, self.uid, self.context
+
+        # Create an employee
+        self.employee_id = self.employee_model.create(
+            cr, uid, {
+                'name': 'Employee 1',
+            }, context=context)
+
+        # Get any existing category
+        self.category_id = self.rule_category_model.search(
+            cr, uid, [], context=context)[0]
+
+        # Create salary rules
+        self.rule_id = self.rule_model.create(
+            cr, uid, {
+                'name': 'Test 1',
+                'sequence': 1,
+                'code': 'TEST_1',
+                'category_id': self.category_id,
+                'appears_on_payslip': True,
+                'active': True,
+                'amount_select': 'fix',
+                'amount_fix': 100,
+                'include_in_payroll_analysis': True,
+            }, context=context
+        )
+        self.rule_2_id = self.rule_model.create(
+            cr, uid, {
+                'name': 'Test 2',
+                'sequence': 2,
+                'code': 'TEST_2',
+                'category_id': self.category_id,
+                'appears_on_payslip': True,
+                'active': True,
+                'amount_select': 'fix',
+                'amount_fix': 200,
+                'include_in_payroll_analysis': False,
+            }, context=context
+        )
+
+        # Create a structure
+        self.structure_id = self.structure_model.create(
+            cr, uid, {
+                'name': 'TEST',
+                'parent_id': False,
+                'code': 'TEST',
+                'rule_ids': [(6, 0, [self.rule_id, self.rule_2_id])]
+            }, context=context
+        )
+
+        # Create a contract for the employee
+        self.contract_id = self.contract_model.create(
+            cr, uid, {
+                'employee_id': self.employee_id,
+                'name': 'Contract 1',
+                'wage': 50000,
+                'struct_id': self.structure_id,
+            }, context=context
+        )
+
+        # Create a payslip
+        self.payslip_ids = [
+            self.payslip_model.create(
+                self.cr, self.uid,
+                {
+                    'employee_id': self.employee_id,
+                    'contract_id': self.contract_id,
+                    'date_from': slip[0],
+                    'date_to': slip[1],
+                    'struct_id': self.structure_id,
+                }, context=context)
+            for slip in [
+                ('2014-01-01', '2014-01-31'),
+                ('2014-02-01', '2014-02-28'),
+            ]
+        ]
+
+        self.payslip_model.compute_sheet(
+            cr, uid, self.payslip_ids, context=context)
+
+    def tearDown(self):
+        cr, uid, context = self.cr, self.uid, self.context
```

Comment:
IMO you can remove the `tearDown`method since the transaction is rolled-back at the end of each test (method).
see https://github.com/odoo/odoo/blob/7.0/openerp/tests/common.py#L85